src/Api/Response/ResponseParser.php

```diff
@@ -198,4 +200,26 @@ protected function parseProducts(\SimpleXMLElement $data)
 
         return $products;
     }
+
+    /**
+     * @param SimpleXMLElement $data
+     * @return array|null
+     */
+    protected function parseSmartDidYouMean(SimpleXMLElement $data)
+    {
+        $originalQuery = $data->query->originalQuery;
```

Comment:
Cast all these values into strings. I'm shocked that this doesn't produce any errors since it returns the XML element, not the actual string value.
Also update the corresponding test, since you will now get strings instead of `SimpleXMLElement`.

Reply:
I think it does not produce any errors, because it automatically casts the value to string when it is used. 
I have updated the pull request based on the comment. Also retested it with the cases as in the screenshots in Youtrack story.